firefox_puppeteer/ui/toolbars.py

```diff
@@ -12,179 +12,187 @@
 
 class NavBar(BaseLib):
     """Provides access to the DOM elements contained in the
-    navigation bar as well as the location bar."""
-
+        navigation bar as well as the location bar."""
+    
```

Comment:
Not sure what's going on with all those doc changes. They introduced a broken indentation across the whole file. Please revert all of those and add the one you need for the go button with the same indentation.